cpp/tests/wholememory_ops/wholememory_embedding_tests.cu

```diff
@@ -238,13 +238,16 @@ TEST_P(WholeMemoryEmbeddingParameterTests, EmbeddingGatherTest)
     wholememory_tensor_description_t embedding_tensor_description;
     wholememory_copy_matrix_desc_to_tensor(&embedding_tensor_description,
                                            &params.embedding_description);
-
+    std::vector<size_t> rank_partition(world_size);
```

Comment:
perhaps we should also allow "random" and "default" partition, just like pytest.